### ISSUE TITLE
Bug in the EXR export compression mode in cli

### ISSUE THREAD

**Issue by Ramel** [darktable-org/darktable#7965]
Hello,

I'm asking myself if it was not a bug in the Exr export compression mode. As I'm trying to export an CR2 to EXR with PIZ compression, but the output file is the same size with compression 4 or 0, approx 300Mo.

`darktable-cli.exe image.cr2 --verbose --out-ext ".exr" --icc-type LIN_REC709 .\ --core --conf plugins/imageio/storage/disk/overwrite=1 --conf plugins/imageio/format/exr/compression=4`

Then if I compress it with magick then the file obtain a compressed size near to 90Mo.

`magick convert image.exr -quiet -compress PIZ image.exr`

It's my Cli? or a bug?

```
Win10
darktable-cli.exe --version
this is darktable-cli 3.4.0
copyright (c) 2012-2020 johannes hanika, tobias ellinghaus
```

Thanks in advance.

Ramel

**Comment by johnny-bit** [comment]
Does the problem persist using UI and export module with same settings?

**Comment by Ramel** [comment]
It's the same, I don't see any Piz compression...

**Comment by Ramel** [comment]
Strange Issue!

**Comment by johnny-bit** [comment]
Hey @Ramel - there might be fix comming! Can you test PR #11001 ? It should help!

**Comment by kmilos** [comment]
> Then if I compress it with magick then the file obtain a compressed size near to 90Mo.

Is it still 32-bit float after this? I somehow doubt it...
